modules/table/src/main/java/org/apache/ignite/internal/table/distributed/SortedIndexLocker.java

```diff
@@ -45,18 +40,8 @@
  * <p>Simply acquires lock on a given row for lookup and remove, acquires lock on a next key for insert.
  */
 public class SortedIndexLocker implements IndexLocker {
-
-    private static final SchemaDescriptor INFINITY_TUPLE_SCHEMA = new SchemaDescriptor(
-            1,
-            new Column[]{
-                    new Column("indexId", NativeTypes.UUID, false),
-                    new Column("partId", NativeTypes.INT32, false)
-            },
-            new Column[0]
-    );
-
     /** Index INF+ value object. */
-    private final BinaryTuple positiveInf;
+    private final Integer positiveInf;
```

Comment:
Well, to be honest, despite the fact that currently it should work because index keys are Bytes, I'll still favor custom InfWrapper here.

Reply:
+1 for custom wrapper

Reply:
Is it enough to use object here, or you want to have a specific class to define infinite infinity?